### ISSUE TITLE
charsets: add more from hyperion

### ISSUE THREAD

**Issue by trufae** [radareorg/radare2#19139]
https://github.com/SDL-Hercules-390/hyperion/blob/master/codepage.c

**Comment by gogo2464** [comment]
Yes. I just update (or not) the doc on charset on r2book and then I will do this charset.
